Cargo.lock

```diff
@@ -1274,7 +1275,7 @@ dependencies = [
  "deno_npm",
  "deno_npm_cache",
  "deno_package_json",
- "deno_path_util",
+ "deno_path_util 0.3.0",
```

Comment:
https://github.com/denoland/deno_doc/pull/684

But I think this is ok to land without because this is a really small dependency

Reply:
There was actually too big of a change in deno_doc for me to do the upgrade. We'll have to do that later.